module6/project/index.md

```diff
@@ -0,0 +1,108 @@
+
+#### TIL
+As part of your project documentation, you will be expected to include a TIL type write-up on your new technology.  This can be added to your TIL repo on GitHub.
+
+## Inception
+
+### Determine MVP
+
+Define the features that will make up your [**Minimum Viable Product**](https://www.agilealliance.org/glossary/mvp/#q=~(infinite~false~filters~(tags~(~'mvp))~searchTerm~'~sort~false~sortDirection~'asc~page~1)).  Your MVP may not be your full application as designed in your Project Pitch, but it should include enough features to meet the Project Requirements.
+
+### Wireframes
+
+* Create a high level [user flow diagram](https://bashooka.com/inspiration/33-excellent-user-flow-examples-for-inspiration/) that lists the steps a user will take when interacting with your application.
+* Make a list of all of the pages required for your application along with what features will exist on each page.
+* For each page, create a [wireframe](https://www.usability.gov/how-to-and-tools/methods/wireframing.html) by sketching out what that user will see when they navigate to that page. Include details such as header text, buttons, drop downs, forms with the specific fields, graphs etc.
+* [Figma](https://www.figma.com/wireframe-tool/?utm_source=google&utm_medium=cpc&utm_campaign=17340391121&utm_term=free%20wireframe%20tools&utm_content=600487515776&gclid=CjwKCAjw5P2aBhAlEiwAAdY7dLtWlIVK0NWWHUXbjj8KMAaBN8P1lO16khOpesJ63utNI5b-cF1e2xoClmoQAvD_BwE) is a great tool for wireframing
+
+### User Stories
+
+* Create User Stories based on your Wireframes
+    * Each wireframe may have multiple user stories.
+* An example user story is:
+    - Feature: User can create a muffin
+    - User story: "As a user, when I visit the muffins index page, I want to click the `new muffin` button so that I can create a new muffin in the application"
+    - Wireframe: `<link to wireframe>`
+    - Subtasks:
+        1. Ensure muffins table is configured with required columns
+        3. Route for muffins index is /muffins
+        3. Route for new muffin form is /muffins/new
+        4. User should be redirected to /muffins after successful muffin creation
+        5. If user is not able to create a muffin, user should see a message that reads `Oops something went wrong!`
+
+At your **third project checkin** you will present your project board and wireframes.
+
+## Development
+
+During the development phase, you will have a few checkins with an instructor. During these checkins be ready to share:
+* The current state of your project board
+* A walkthrough of your project in it's current state
+* Any blockers you are experiencing.
+
+## Presentation/Documentation
+
+The last full day of the project will be dedicated to documentation, and preparing a presentation for your project.
```

Comment:
I think we could remove the link to the Demo Guide and just say that at the end of Mod 6 you will be giving a presentation to demo the feature you added to your capstone project. 

I think we still do "Presentations should be about 5 minutes long (and definitely not more than 8!). "

And as the last level of gradual release I think it's totally find to not have a demo guide for this presenation.